Formula/pkg-config.rb

```diff
@@ -20,6 +20,8 @@ class PkgConfig < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "3d9b8bf9b7b4bd08086be1104e3e18afb1c437dfaca03e6e7df8f2710b9c1c1a"
   end
 
+  conflicts_with "pkgconf", because: "both install `pkg.m4`"
```

Comment:
Let's add this in a separate PR after merging this one so we're not forced to rebuilt bottles for `pkg-config`.